internal/test/issues/issue-1093/child.cfg.yaml

```diff
@@ -0,0 +1,10 @@
+# child.yaml
+package: api
+generate:
+  gin-server: true
+  embedded-spec: true
+  strict-server: true
+  models: true
+import-mapping:
+  parent.api.yaml: github.com/deepmap/oapi-codegen/internal/test/issues/issue-1093/api/parent
+output: ../../examples/import-mapping/api/child/child.gen.go
```

Comment:
oh these config values are stale from when I had the test in the wrong spot. Let me fix that.